converter/output/rita/streaming/dates/rita_dates_test.go

```diff
@@ -727,3 +728,108 @@ func generateNSessions(n int64, targetSessionEnd time.Time) []session.Aggregate
 	}
 	return sessions
 }
+
+func TestStrobes(t *testing.T) {
+	fixtures := fixtureManager.BeginTest(t)
+	defer fixtureManager.EndTest(t)
+
+	env := fixtures.Get(integrationtest.EnvironmentFixture.Key).(environment.Environment)
+	strobeLimit := env.GetOutputConfig().GetRITAConfig().GetStrobe().GetConnectionLimit()
+	expectedConnCount := strobeLimit + 5
```

Comment:
Essentially, we push through the ConnectionLimit + 5 connections with the same source and destination through the streaming session writer and ensure that the conn collection is empty and that the freqconn collection contains a valid record for the data.